frontend/src/components/browse-sightings-page/SightingCard.tsx

```diff
@@ -11,8 +11,19 @@ export const SightingCard: React.FunctionComponent<SightingProps> = ({
 }) => {
   return (
     <div className="sighting-card">
-      <h3 className="fade-in">Sighting #{sighting.id}</h3>
+      {!sighting.species?.name ? (
+        <h3 className="sighting-card_title fade-in">
+          Sighting recorded by {sighting.seenBy}
+        </h3>
+      ) : (
+        <h3 className="sighting-card_title fade-in">
+          Sighting of {sighting.species?.name}
+        </h3>
+      )}
 
+      <h6 className="sighting-card_date fade-in">
+        {new Date(sighting.seenOn).toDateString()}
+      </h6>
```

Comment:
This isn't (semantically) a heading - use a `<p>` and style it
```suggestion
      <p className="sighting-card_date fade-in">
        {new Date(sighting.seenOn).toDateString()}
      </p>
```

Reply:
done, thanks